### ISSUE TITLE
Deprecated dependencies

### ISSUE THREAD

**Issue by aravindvnair99** [BePo65/license-report#58]
```console
$ npm i -g license-report@latest
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated stubborn@1.2.5: no longer maintained
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

**Comment by BePo65** [comment]
thank you for this information. In the end it all boils down to `request` being phased out. The author tells us that there is no urgent need to change to another package, but it should be done anyway.

I prepared an update refactoring license-report to async-await and also replacing 'request' with 'got'. So I hope that we can solve this issue in one of the next releases.

**Comment by BePo65** [comment]
will be solved by pr #56

**Comment by BePo65** [comment]
Fixed in next release based on pr #65.
